service/controller/resource/namespaceauth/create.go

```diff
@@ -29,7 +29,7 @@ func (r *Resource) EnsureCreated(ctx context.Context, obj interface{}) error {
 	}
 	tenantAdminRole := role{
 		name:        "tenant-admin",
-		targetGroup: r.namespaceAuth.ViewAllTargetGroup,
+		targetGroup: r.namespaceAuth.TenantAdminTargetGroup,
```

Comment:
Bug? Should be mentioned in the changelog?

Reply:
done